libr/anal/flirt.c

```diff
@@ -794,7 +794,7 @@ static ut8 read_module_referenced_functions(RFlirtModule *module, RBuffer *b) {
 				goto err_exit;
 			}
 		}
-		if ((int) ref_function_name_length < 0) {
+		if ((int) ref_function_name_length < 0 || ref_function_name_length > R_FLIRT_NAME_MAX) {
```

Comment:
Why `>=` ? 

https://github.com/radare/radare2/blob/ee1727f272d69d904cc84e4d368ece8c260f5ead/libr/anal/flirt.c#L800

Reply:
Because at https://github.com/radare/radare2/pull/11300/files#diff-a7c93b8aab99f805c93479a733cb7decR810 you would end up writing to `name[R_FLIRT_NAME_MAX]`, which is an array of R_FLIRT_NAME_MAX bytes, so you would overflow one byte.